src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -1922,8 +1922,16 @@ HRESULT CEndlessUsbToolDlg::OnInstallDualBootClicked(IHTMLElement* pElement)
 
 	if (UpdateDualBootTexts()) {
 		m_selectedInstallMethod = InstallMethod_t::UninstallEndless;
+		eventTracking(_T("Selected"));
```

Comment:
Controversial suggestion: get rid of the standalone path, just hide the "advanced options" bit at the bottom of the screen when run as `endless-uninstaller.exe`.